src/Illuminate/Collections/Enumerable.php

```diff
@@ -32,9 +32,10 @@ public static function times($number, callable $callback = null);
      *
      * @param  int  $from
      * @param  int  $to
+     * @param  int  $step
      * @return static
      */
-    public static function range($from, $to);
+    public static function range($from, $to, $step = 1);
```

Comment:
breaking change

Reply:
@GrahamCampbell  that's why this is targetting master